_includes/common-modal.html

```diff
@@ -0,0 +1,18 @@
+<div class="modal fade" id="{{include.id}}" tabindex="-1" role="dialog">
```

Comment:
@francho Just try not to omit the required spaces in variable anchors: `{{ include.id }}` (there are a lot of cases like this)